### PR TITLE
ArduPlane Feature added.

### PR DESCRIPTION
**Pull request by bahadirbuber**
Arduplane flight modes added.

### CODE REVIEW

**Comment by JonasVautherin** [comment]
Looks promising! Failing some tests though.

Also do not forget to fix the style before pushing, with `./tools/fix_style.sh .` :innocent:

File: src/mavsdk/core/flight_mode.h [hunk 1]
```diff
@@ -7,6 +7,9 @@ namespace mavsdk {
 
 enum class FlightMode {
     Unknown,
+    FBWA,
```
Comment (julianoes) [inline src/mavsdk/core/flight_mode.h:10]:
```suggestion
    Fbwa,
```

File: src/mavsdk/core/system_impl.cpp [hunk 2]
```diff
@@ -844,6 +844,16 @@ SystemImpl::make_command_ardupilot_mode(FlightMode flight_mode, uint8_t componen
                     static_cast<float>(flight_mode_to_ardupilot_rover_mode(flight_mode));
             }
             break;
+        case MAV_TYPE::MAV_TYPE_FIXED_WING:
+            if (flight_mode_to_ardupilot_plane_mode(flight_mode) == ardupilot::PlaneMode::Unknown) {
+                LogErr() << "Cannot translate flight mode to ardupilot pplane mode.";
```
Comment (julianoes) [inline src/mavsdk/core/system_impl.cpp:849]:
```suggestion
                LogErr() << "Cannot translate flight mode to ArduPlane mode.";
```

File: src/mavsdk/core/flight_mode.h (hunk 1 above)
Comment (julianoes) [reply src/mavsdk/core/flight_mode.h:10]: And I'd probably add the new modes at the bottom rather than the top, so we don't change the existing indices.

**Review by julianoes — COMMENTED** [review]
Thanks for the PR. See inline comments.

**Comment by julianoes** [comment]
I've added a commit to clean this up and fix style according to the MAVSDK conventions.

**Review by JonasVautherin — APPROVED** [review]
Great! :rocket:

**Pull request merged**